lib/skeleton/config/scripts/ios/devices_local.txt

```diff
@@ -0,0 +1,4 @@
+# <DEVICE_TARGET (UUID)> | <DEVICE_ENDPOINT (IP)> | <DEVICE_NAME> | <DEVICE_TYPE> (Simulator or Device)
```

Comment:
Is this file really necessary? A column in the devices.txt could solve your problem?